tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/connectors.py

```diff
@@ -446,3 +452,60 @@ def list(
 
     console.print(table)
     return True
+
+
+@connectors.command(cls=DaggerPipelineCommand, help="Autoformat connector code.")
+@click.pass_context
+def format(ctx: click.Context) -> bool:
+    if ctx.obj["is_ci"] and get_last_commit_message().startswith("🤖"):
```

Comment:
Couple things. 

1. Why do we skip?
2. Do we want to increase this to a list of known bot authors like octavia?

Reply:
We skip to avoid an infinite loop of triggers 😄 
And I also believe bot generated code should not be formatted, it should be by design.

Reply:
Make sense. Only nit I would add is having a fall back that checks if the author is one of our known bots.

Just in case one of our bots doesnt use the emoji

Reply:
Just recommending here as I do think this would be a good idea. Even if its as simple as
```python
def is_bot_commit(commit):
    return get_last_commit_message().startswith("🤖") or "octavia" in get_commit_author()
```